pkg/controller/clusterstate/cache/cache.go

```diff
@@ -139,10 +98,7 @@ func newClusterStateCache(config *rest.Config) *ClusterStateCache {
 
 func (sc *ClusterStateCache) Run(stopCh <-chan struct{}) {
 	klog.V(8).Infof("Cluster State Cache started.")
-
-	go sc.podInformer.Informer().Run(stopCh)
```

Comment:
I'm sure you checked this, but we have an informer for pods somewhere else, right? Like to keep track of how many are alive before recycling the job.